### ISSUE TITLE
Term Request: Spiky corneal stromal opacities

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11746]
**Preferred term label:**
Spiky corneal stromal opacities

**Synonyms**
* Star-shaped corneal stromal deposits
* Icicle-shaped corneal opacities
* Spider-shaped corneal stromal deposits

**Definition (free text, please give PubMed ID)**
Branching, thorn-like, icicle-shaped, or spider-shaped discrete opacities in the superficial to mid-stroma of the cornea, typically displaying translucent centers under retroillumination and composed of a mixture of keratohyalin and amyloid deposits (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Opacification of the corneal stroma (HP:0007759)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Granular corneal dystrophy, type 2 (GCD2 / Avellino corneal dystrophy) (ORPHA:98963, OMIM:607541)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
